src/main/java/seedu/address/storage/Storage.java

```diff
@@ -26,6 +26,11 @@
     @Override
     Optional<ReadOnlyAddressBook> readAddressBook() throws DataConversionException, IOException;
 
+    @Override
+    default Optional<ReadOnlyAddressBook> readAddressBook(Path filePath) throws DataConversionException, IOException {
+        return Optional.empty();
+    }
+
```

Comment:
May I know what is the reason for adding these?

Reply:
I think this was some sort of mistake, i got rid of the dufault and the return statement.